src/frontend/web_application/src/styles/object/o-form-element.scss

```diff
@@ -2,6 +2,7 @@
   display: inline-block;
   padding: ($co-margin / 2);
   border: 0;
+  background-color: transparent;
```

Comment:
I checked and it seems ok. background-color wasn't defined here, it is only called with modifiers (o-form-element--dark and o-form-element--light)